src/PaymentStates.php

```diff
@@ -30,4 +30,5 @@ final class PaymentStates
     public const STATE_FAILED = 'failed';
     public const STATE_REFUNDED = 'refunded';
     public const STATE_PARTIALLY_REFUNDED = 'partially_refunded';
+    public const STATE_REFUND_CARD_SCHEME_FAILED = 'refund_card_scheme_failed';
```

Comment:
Why should we use the "scheme" word here?

Reply:
+1

Reply:
I mention scheme to make it a little more clear at which point the refund failed, which based on the [docs],(https://docs.adyen.com/online-payments/refund?tab=successful_refund_request_1#refund-failed) this happens at scheme level. It is then up to the plugin if it wants to set the order to some specific `failed` status, or to a more detailed one.

Reply:
Resolved in https://github.com/Adyen/php-webhook-module/pull/3/commits/29764ae4dadf4c9c3879218a087f0000f1a9f559